File: JPAGamingHub/src/main/java/com/skilldistillery/gaminghub/entities/MeetUp.java
package com.skilldistillery.gaminghub.entities;

import java.time.LocalDate;
import java.time.LocalDateTime;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "meetup")
public class MeetUp {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	
	private LocalDate date;
	private String timezone;
	private Integer capacity;
	private String description;
	private LocalDateTime created;
	private LocalDateTime updated;

}

File: ngGamingTavern/src/app/models/meetup/meetup.ts
export class Meetup {
  id: number;
  enabled: boolean;
  name: string;
  date: string;
  capacity: number;
  description: string;
  created: string;
  updated: string;

  constructor(
    id: number = 0,
    enabled: boolean,
    name: string = "",
    date: string = "",
    capacity: number = 0,
    description: string = "",
    created: string,
    updated: string
  ) {
    this.id = id;
    this.enabled = enabled;
    this.name = name;
    this.date = date;
    this.capacity = capacity;
    this.description = description;
    this.created = created;
    this.updated = updated;
  }
}
